pkg/segment/writer/segwriter.go

```diff
@@ -334,7 +334,7 @@ func (ss *SegStore) doLogEventFilling(ple *ParsedLogEvent, tsKey *string) (bool,
 		colWip, _, matchedCol = ss.initAndBackFillColumn(cname, SS_DTYPE(ctype), matchedCol)
 
 		switch ctype {
-		case VALTYPE_ENC_SMALL_STRING[0]:
+		case VALTYPE_ENC_SMALL_STRING[0], VALTYPE_ENC_BOOL[0]:
```

Comment:
Can we really do this? I though we encode strings as their type (string), then length, then value; and for bools just their type (bool) and value. So I think this will cause issues with ingesting booleans. Are you absolutely sure it will not?

Reply:
yeah this is not right, we would be copying some garbage for bool types.

Reply:
fixed it in a separate PR